### PR TITLE
Feature porting over react components

### PR DESCRIPTION
**Pull request by mkanoria**
Porting over FE components + fixing styling on Login and Events page

Using the standardized Modal component, added `next-images` for svgs 

image

image

image

### CODE REVIEW

File: components/Cards.js [hunk 1]
```diff
@@ -45,7 +45,9 @@ function EventCard({ event, isAdmin, openEditModal }) {
           </span>
 
           {/* Event number of performances */}
-          <span>{event.num_performances} Performances</span>
+          <span>
+            {event.num_performances ? event.num_performances : 0} Performances
```
Comment (OustanDing) [inline components/Cards.js:49]:
```suggestion
            {event.num_performances || 0} Performances
```

Comment (OustanDing) [reply components/Cards.js:49]: Shorter, not a big deal though.

File: pages/api/events/get.js [hunk 2]
```diff
@@ -9,13 +9,15 @@ export default async (req, res) => {
   if (session) {
     // Collect eventID from URL query
     const { id: eventID } = req.query;
-
+    console.log(req.query);
+    console.log(eventID);
     // If eventID is provided
     if (eventID) {
       try {
+        console.log("In here");
```
Comment (OustanDing) [inline pages/api/events/get.js:17]: We should remove these `console.log`s?

File: pages/api/events/get.js [hunk 3]
```diff
@@ -9,13 +9,15 @@ export default async (req, res) => {
...
         // Collect event with eventID
         let event = await getEventByID(eventID);
-
+        console.log(event);
```
Comment (OustanDing) [inline pages/api/events/get.js:20]: Same here.

File: styles/DatePicker.css [hunk 4]
```diff
@@ -0,0 +1,164 @@
+/*
```
Comment (OustanDing) [inline styles/DatePicker.css:1]: Should we make this a SASS file?

**Review by OustanDing — APPROVED** [review]
LGTM. A few small comments.

Can we change some of the file names to be singular (eg. `Buttons` --> `Button`) now that we have the reusable components?

File: pages/api/events/get.js (hunk 2 above)
Comment (mkanoria) [reply pages/api/events/get.js:17]: Good catch LMAO

**Pull request merged**